src/problems/MetalAdvectionProblem/metal_uniform.in

```diff
@@ -0,0 +1,39 @@
+# *****************************************************************
+# Problem size and geometry
+# *****************************************************************
+geometry.prob_lo     =  0.0       0.0       -1.2072e+22
+geometry.prob_hi     =  3.018e21  3.018e21   1.2072e+22
+geometry.is_periodic =  1    1    0
+
+# *****************************************************************
+# VERBOSITY
+# *****************************************************************
+amr.v              = 1      # verbosity in Amr
+
+# *****************************************************************
+# Resolution and refinement
+# *****************************************************************
+amr.n_cell          = 128  128 1024
+amr.max_level       = 0     # number of levels = max_level + 1
+amr.max_grid_size   = 128   # at least 128 for GPUs
+amr.blocking_factor = 32   # grid size must be divisible by this
+amr.n_error_buf     = 3     # minimum 3 cell buffer around tagged cells
+amr.grid_eff        = 0.7   # default
+amrex.use_gpu_aware_mpi=0
```

Comment:
Why is GPU-aware MPI disabled? It should work on Setonix.

Reply:
Ah so, this is an outdated version of the input file which I am not using. I'll just delete this file.

Reply:
Can you put an updated input file in the PR? I still need one to run on Frontier, and if we add a regression test, an input file is needed for that.